### PR TITLE
Update retention policies docs

### PR DESCRIPTION
**Pull request by N-lson**
[sc-84718]

### CODE REVIEW

File: src/pages/docs/administration/retention-policies/retention-policy-tentacle-cleanup-and-troubleshooting.md [hunk 1]
```diff
@@ -115,26 +116,22 @@ See below the messages you will have in your raw deployment logs at the end of a
 You can find your packages in the `<Tentacle Home>\Files` directory.
 or `<Tentacle Home>\[Instance Name])\Files` if you have multiple 
 Tentacle instances on one machine.
-- For Windows Tentacles, the default directory is: `C:\Octopus\Files`
```
Comment (shaun-od) [inline src/pages/docs/administration/retention-policies/retention-policy-tentacle-cleanup-and-troubleshooting.md:118]: Why was this removed?

**Review by shaun-od — COMMENTED** [review]
I have one questions about the removal of the default locations before I give it the thumbs up otherwise looks good to me.

File: src/pages/docs/administration/retention-policies/retention-policy-tentacle-cleanup-and-troubleshooting.md (hunk 1 above)
Comment (N-lson) [reply src/pages/docs/administration/retention-policies/retention-policy-tentacle-cleanup-and-troubleshooting.md:118]: It's already mentioned in lines 21-26 (doesn't show up in the PR since I didn't change those lines).

**Review by shaun-od — APPROVED** [review]
LGTM
